src/constants/WINNING_PRICE.js

```diff
@@ -0,0 +1,9 @@
+const WINNING_PRICE = {
+  6: 2000000000,
+  "5+1": 30000000,
+  5: 1500000,
+  4: 50000,
+  3: 5000,
+};
```

Comment:
큰 숫자를 사용할 때는 numeric separator를 사용해주시면 좋을거 같아요
```suggestion
const WINNING_PRICE = {
  6: 2_000_000_000,
  "5+1": 30_000_000,
  5: 1_500_000,
  4: 50_000,
  3: 5_000,
};
```

Reply:
6 ~ 3 같은 의미를 알기 어려운 숫자들도 상수로 만들어보시면 좋겠네요

Reply:
numeric separator 적용해보겠습니다!
6~3 같은 경우도 의미있는 상수로 변경하겠습니다!

Reply:
#### 1. `WINNING_PRICE` -> numeric separator로 변경 ([8d5ecbb](https://github.com/woowacourse/javascript-lotto/pull/364/commits/8d5ecbba24fbe1f07d66c176a610f54de15ed659))

Reply:
#### 2. 3~6 까지 로또 체크 갯수 상수화 ([9154a75](https://github.com/woowacourse/javascript-lotto/pull/364/commits/9154a750537ec92e50b2e54769b4479ec5b9081a))
말씀하신대로 매치되는 갯수들 상수화를 진행했습니다. 어떤 식으로 상수화를 하면 좋을까 고민하면서 초반에는 그냥 쌩 string 값으로 `MATCH_SIX`로 적용을 했습니다. 하지만, 생각보다 이 값을 참조하는 곳이 많았습니다.
예를 들면, 1. LottoPack 객체에서 실제 매치된 개수를 체크하는 로직 / 2. 당첨 통계를 출력할 때 outputView에서 사용하는 메시지 / 3. 당첨 금액을 관리하는 WinningPrice 와 같이 여러 곳에서 동일한 값을 사용하다 보니, 공통된 상수를 정의하는 것이 필요하다고 생각했습니다.
저는 아래와 같은 방식으로 상수화를 진행했습니다.
```
export const MATCH_COUNT = Object.freeze({
  THREE: "3개 일치 (5,000원)",
  FOUR: "4개 일치 (50,000원)",
  FIVE: "5개 일치 (1,500,000원)",
  FIVE_BONUS: "5개 일치, 보너스 볼 일치 (30,000,000원)",
  SIX: "6개 일치 (2,000,000,000원)",
});
```
이렇게 상수화를 진행한다면 LottoPack, outputView, WinningPrice 등 여러 곳에서 일관되게 `MATCH_COUNT`만 참조가 가능하고, 가독성 측면에서도 굉장히 좋다고 생각합니다. 또한 outputView에서 당첨통계를 보여주던 지저분한 ui를 깔끔하게 반복문으로 처리가 가능합니다.
하지만 문자열이 길어지면서 메모리 사용량이 증가할 것 같은데, 이 방식에 대해서 어떻게 생각하시는지 궁금합니다.

Reply:
좋은 방법으로 보이네용 ㅎㅎ 👍 메모리 사용량이 증가하면 어떤 문제가 발생할 수 있나요? 지금은 크게 고민하지 않으셔도 될 부분이라고 생각합니다 ㅎㅎ

Reply:
음, 맞습니다!. 개인적으로 프로그램이 느려지지는 않을까?라는 고민이 되어 여쭤봤습니다. 하지만 프로그램 자체가 크지 않기 때문에 지금 당장 고민할 부분은 아닌 것 같습니다.